lms/static/certificates/sass/_views.scss

```diff
@@ -11,3 +11,35 @@
 // #DISTINCTION CERT
 // ------------------------------
 
+
+// ------------------------------
+// #INVALID
+// ------------------------------
+
+.wrapper-content.status-invalid {
+    @extend %layout-wrapper;
+
+    .wrapper-content-grid {
+        @extend %layout;
+    }
+
+  .content-main {
+    margin-bottom: spacing-vertical(base);
+
+    @include susy-breakpoint($bp-medium, $susy) {
+      @include span(9 of 12 first);
+    }
+  }
+
+  .content-secondary {
+    margin-top: spacing-vertical(x-small);
+
+    @include susy-breakpoint($bp-medium, $susy) {
+      @include span(3 of 12 last);
+    }
+
+    .about-title {
+      @extend .hd-6;
```

Comment:
NIT: can you extend the Sass placeholder (``%hd-6``) rather than the utility class?